### DISCUSSION TITLE
[Flex layout] probably justify-* utils should be like we use in plain css, justify-content-*

### DISCUSSION THREAD

**Discussion by dipeshwalia** [tailwindlabs/tailwindcss#2383] [Ideas]
👋 Hi tailwind community, 

I am exploring tailwindcss in my regular workflows, I notice justify-content utils are under justify namespace. 

It seems confusing to me, since my default mental model for flex suggest me using justify-content itself. 

Am i missing something ?

**Comment by adamwathan** [comment]
We kept the names as short as possible (so `justify-between` instead of `justify-content-space-between`) because the utilities are really common and every character counts when trying to avoid things getting ridiculously verbose 👍 

At this point they have been named that way for 3 years and changing them would break thousands of websites so no plans to change them I don't think.
